pkg/kubelet/cadvisor.go

```diff
@@ -59,7 +59,7 @@ func (kl *Kubelet) GetContainerInfo(podFullName string, uid types.UID, container
 	if cc == nil {
 		return nil, nil
 	}
-	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.dockerClient, false)
+	dockerContainers, err := dockertools.GetKubeletDockerContainers(kl.containerRuntime, false)
```

Comment:
It doesn't look to me that docker is going to support ACI.  Does that mean that we need to revisit this?  Or is it sufficient to say that we'll inject a mux here and that the Pod spec has enough information to route to the appropriate concrete implementation?